### PR TITLE
prepare v2.24.0 for falcosidekick

### PR DESCRIPTION
**Pull request by developer-guy**
**What type of PR is this?**

/kind documentation

/kind feature

/kind chart-release

**Any specific area of the project related to this PR?**

/area falcosidekick-chart

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] CHANGELOG.md updated

### CODE REVIEW

File: falcosidekick/templates/secrets.yaml [hunk 1]
```diff
@@ -227,6 +227,41 @@ data:
   WAVEFRONT_METRICNAME: "{{ .Values.config.wavefront.metricname | b64enc}}"
   WAVEFRONT_MINIMUMPRIORITY: "{{ .Values.config.wavefront.minimumpriority | b64enc}}"
 
+  # Grafana Output
+  GRAFANA_HOSTPORT: "{{ .Values.config.grafana.hostport | b64enc}}"
+  GRAFANA_APIKEY: "{{ .Values.config.grafana.apikey | b64enc}}"
+  GRAFANA_DASHBOARDID: "{{ .Values.config.grafana.dashboardid | b64enc}}"
+  GRAFANA_PANELID: "{{ .Values.config.grafana.panelid | b64enc}}"
+  GRAFANA_ALLFIELDSASTAGS: "{{ .Values.config.grafana.panelid | printf "%t" | b64enc}}"
```
Comment (Issif) [inline falcosidekick/templates/secrets.yaml:235]:
```suggestion
  GRAFANA_ALLFIELDSASTAGS: "{{ .Values.config.grafana.allfieldsastags | printf "%t" | b64enc}}"
```

File: falcosidekick/templates/secrets.yaml [hunk 2]
```diff
@@ -227,6 +227,41 @@ data:
   WAVEFRONT_METRICNAME: "{{ .Values.config.wavefront.metricname | b64enc}}"
   WAVEFRONT_MINIMUMPRIORITY: "{{ .Values.config.wavefront.minimumpriority | b64enc}}"
 
+  # Grafana Output
+  GRAFANA_HOSTPORT: "{{ .Values.config.grafana.hostport | b64enc}}"
+  GRAFANA_APIKEY: "{{ .Values.config.grafana.apikey | b64enc}}"
+  GRAFANA_DASHBOARDID: "{{ .Values.config.grafana.dashboardid | b64enc}}"
+  GRAFANA_PANELID: "{{ .Values.config.grafana.panelid | b64enc}}"
```
Comment (Issif) [inline falcosidekick/templates/secrets.yaml:234]:
```suggestion
  GRAFANA_DASHBOARDID: "{{ .Values.config.grafana.dashboardid | toString | b64enc}}"
  GRAFANA_PANELID: "{{ .Values.config.grafana.panelid | toString | b64enc}}"
```

File: falcosidekick/values.yaml [hunk 3]
```diff
@@ -267,6 +267,42 @@ config:
...
+
+  yandex:
+    accesskeyid: ""  # yandex access key
+    secretaccesskey: ""  # yandex secret access key
+    region: ""  # yandex storage region (default: ru-central-1)
+  s3:
+    endpoint: ""  # yandex storage endpoint (default: https://storage.yandexcloud.net) 
+    bucket: "falcosidekick"  # Yandex storage, bucket name
+    prefix: ""  # name of prefix, keys will have format: s3://<bucket>/<prefix>/YYYY-MM-DD/YYYY-MM-DDTHH:mm:ss.s+01:00.json
+    minimumpriority: ""  # minimum priority of event for using this output, order is emergency|alert|critical|erro
```
Comment (Issif) [inline falcosidekick/values.yaml:297]:
```suggestion
    s3:
      endpoint: ""  # yandex storage endpoint (default: https://storage.yandexcloud.net) 
      bucket: "falcosidekick"  # Yandex storage, bucket name
      prefix: ""  # name of prefix, keys will have format: s3://<bucket>/<prefix>/YYYY-MM-DD/YYYY-MM-DDTHH:mm:ss.s+01:00.json
      minimumpriority: ""  # minimum priority of event for using this output, order is emergency|alert|critical|erro
```

**Review by Issif — CHANGES_REQUESTED** [review]

File: falcosidekick/templates/secrets.yaml (hunk 1 above)
Comment (developer-guy) [reply falcosidekick/templates/secrets.yaml:235]: thank you so much !!!

File: falcosidekick/templates/secrets.yaml (hunk 2 above)
Comment (developer-guy) [reply falcosidekick/templates/secrets.yaml:234]: thank you so much !!!

File: falcosidekick/values.yaml (hunk 3 above)
Comment (developer-guy) [reply falcosidekick/values.yaml:297]: thank you so much !!!

File: falcosidekick/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,20 @@ numbering uses [semantic versioning](http://semver.org).
...
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.3.14
+
+### Major Changes
+
+* New output: Grafana (PR#254)
+* New output: Fission (PR#255 thanks to @gauravgahlot)
+* New output: Yandex Cloud S3 (PR#261 thanks to @nar3k)
+* New output: Kafka REST (PR#263 thanks to @dirien)
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:15]:
```suggestion
* Add `Grafana` output
* Add `Fission` output
* Add `Yandex Cloud S3` output
* Add `Kafka REST` output
```

File: falcosidekick/CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,20 @@ numbering uses [semantic versioning](http://semver.org).
...
+
+### Minor changes
+
+* Set header x-amz-acl to bucket-owner-full-control for output AWS S3 (PR#264 thanks to @Kaizhe)
+* Docker image is now available on AWS ECR Public Gallery (PR#265 thanks to @maxgio92)
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:20]:
```suggestion
* Docker image is now available on AWS ECR Public Gallery (`--set image.registry=public.ecr.aws`
```

**Review by Issif — CHANGES_REQUESTED** [review]
You forgot to complete the table with all values in README 🙏

File: falcosidekick/Chart.yaml [hunk 6]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: 2.23.1
+appVersion: 2.24.0
 description: A simple daemon to help you with falco's outputs
```
Comment (Issif) [inline falcosidekick/Chart.yaml:3]:
```suggestion
description: Connect Falco to your ecosystem
```

I prefer this description now

Comment (developer-guy) [reply falcosidekick/Chart.yaml]: fixed

File: falcosidekick/CHANGELOG.md (hunk 5 above)
Comment (developer-guy) [reply falcosidekick/CHANGELOG.md]: fixed

File: falcosidekick/CHANGELOG.md (hunk 4 above)
Comment (developer-guy) [reply falcosidekick/CHANGELOG.md]: fixed

File: falcosidekick/CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,19 @@ numbering uses [semantic versioning](http://semver.org).
...
+### Major Changes
+
+* Add `Grafana` output
+* Add `Fission` output
+* Add `Yandex Cloud S3` output
+* Add `Kafka REST` output
+
+### Minor changes
+
+* Docker image is now available on AWS ECR Public Gallery (`--set image.registry=public.ecr.aws`
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:19]:
```suggestion
* Docker image is now available on AWS ECR Public Gallery (`--set image.registry=public.ecr.aws`)
```

**Review by Issif — CHANGES_REQUESTED** [review]

File: falcosidekick/CHANGELOG.md (hunk 7 above)
Comment (developer-guy) [reply falcosidekick/CHANGELOG.md:19]: fix

File: falcosidekick/README.md [hunk 8]
```diff
@@ -291,13 +295,40 @@ The following table lists the main configurable parameters of the Falcosidekick
 | `config.rabbitmq.queue`                     | Rabbitmq Queue name                                                                                                                          |                                                                                                   |
 | `config.rabbitmq.minimumpriority`               | minimum priority of event for using use this output, order is `emergency\|alert\|critical\|error\|warning\|notice\|informational\|debug or ""`                                                | `debug`                                                                                           |
 | `config.wavefront.endpointtype`   | Wavefront endpoint type, must be 'direct' or 'proxy'. If not empty, with endpointhost, Wavefront output is enabled
-| `config.wavefront.endpointhost`   | Wavefront endpoint address (only the host). If not empty, with endpointhost, Wavefront output is enabled
+| `config.wavefront.endpointhost`   | Wavefront endpßßßßoint address (only the host). If not empty, with endpointhost, Wavefront output is enabled
```
Comment (Issif) [inline falcosidekick/README.md:298]:
```suggestion
| `config.wavefront.endpointhost`   | Wavefront endpoint address (only the host). If not empty, with endpointhost, Wavefront output is enabled
```

Comment (developer-guy) [reply falcosidekick/README.md]: 🤦🏻‍♂️ fixed

**Review by Issif — APPROVED** [review]

**Pull request merged**